llmfoundry/callbacks/curriculum_learning_callback.py

```diff
@@ -57,8 +59,8 @@ class CurriculumLearning(CallbackWithConfig):
             being used. Note that this is the full train config and must
             contain the 'train_loader', 'device_train_batch_size', and
             'tokenizer' keys.
-        duration (Union[Time, str, int], optional): The duration of the first datamix
-            (which corresponds to the train_loader). Defaults to None.
+        duration (Union[Time, str, int]): The duration of the first datamix
```

Comment:
why does this default change?

Reply:
Changing the API for the dataclass so cleaning up some defaults